libcodechecker/libhandlers/check.py

```diff
@@ -453,21 +382,21 @@ def __update_if_key_exists(source, target, key):
             logfile=[logfile],
             output_path=report_dir,
             output_format='plist',
-            jobs=args.jobs,
-            add_compiler_defaults=args.add_compiler_defaults
+            jobs=args.jobs
         )
         # Some arguments don't have default values.
         # We can't set these keys to None because it would result in an error
         # after the call.
         args_to_update = ['skipfile',
                           'analyzers',
+                          'add_compiler_defaults',
```

Comment:
Why can't this have a default value?

Reply:
As above. Boolean options' help messages usually look better if `default: False` is not given. The code behaves in a way that if it is not passed, it is considered `False`.